sdk/translation/Azure.AI.Translation.Text/samples/Sample2_Translate.md

```diff
@@ -0,0 +1,366 @@
+# Translate
+
+## Create a `TextTranslationClient`
+
+To create a new `TextTranslationClient`, you will need the service endpoint and credentials of your Translator resource. In this sample, however, you will use an `AzureKeyCredential` and region, which you can create with an API key.
+
+```C#
+AzureKeyCredential credential = new("<apiKey>");
+TextTranslationClient client = new(credential, "<region>");
+```
+
+The values of the `apiKey` and `region` variables can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
+
+### Translate text
+Translate text from known source language to target language.
+```C#
+try
+{
+    string from = "en";
+    string targetLanguage = "cs";
+    string inputText = "This is a test.";
+
+    Response<IReadOnlyList<TranslatedTextItem>> response = await client.TranslateAsync(targetLanguage, inputText, from: from).ConfigureAwait(false);
+    IReadOnlyList<TranslatedTextItem> translations = response.Value;
+    TranslatedTextElement translation = translations.FirstOrDefault();
```

Comment:
Is this what we expect actual programs to do? Or users should select the element with the highest score?

Reply:
It really depends on the use case. I don't think there is only one / main use-case.